build.sbt

```diff
@@ -126,6 +127,18 @@ lazy val fansiVersion = Def.setting {
   else "0.2.9"
 }
 
+lazy val fs2Version = Def.setting {
+  if (scalaVersion.value.startsWith("2.11")) "2.1.0"
+  else if (scalaVersion.value == "3.0.0-M2") "2.5.0"
+  else "2.5.3"
+}
+
+lazy val munitVersion = Def.setting {
+  if (scalaVersion.value == "3.0.0-M2")
+    "0.7.21"
+  else V.munit
+}
+
```

Comment:
Previously tests weren't running on Scala 3, so we didn't have to cherry pick versions.